scss/_ps-dropdown.scss

```diff
@@ -0,0 +1,64 @@
+.ps-dropdown {
+    font-size: 1em;
+    padding: .188em 0;
+    background: #fff;
+    width: 100%;
+    line-height: 2.286em;
+    cursor: pointer;
+    box-shadow: 0 0 0 0 rgba(0, 0, 0, 0);
+    transition: .15s ease-out;
+
+    &.bordered {
+      border: $gray-300 1px solid;
+      border-radius: 4px;
+
+      &.show {
+        border-bottom-left-radius: 0;
+        border-bottom-right-radius: 0;
+        box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.1);
+      }
+    }
+
+    .dropdown-label {
+      padding: 0 5px 0 15px;
+      flex-grow: 1;
+    }
+
+    .arrow-down{
```

Comment:
```suggestion
    .arrow-down {
```

Reply:
This will be autofixed after the merge, stylelint has been merge after some PRs there

Reply:
What do you mean, autofixed? You still have to run the fixer yourself.